server/src/controllers/contactController.ts

```diff
@@ -49,6 +49,26 @@ contactController.use(checkScopes);
  */
 contactController.post('/', async (req: Request, res: Response) => {
   try {
+    const contact = await prisma.contacts.create({
+      data: {
+        firstname: req.body.firstname,
+        lastname: req.body.lastname,
+        email: req.body.email,
+        phone: req.body.phone,
+        address: req.body.address,
+        city: req.body.city,
+        state: req.body.state,
+        country: req.body.country,
+        postalcode: req.body.postalcode,
+        donorbadge: req.body.donorbadge,
+        seatingaccom: req.body.seatingaccom,
+        comments: req.body.comments,
+        visitsource: req.body.visitsource,
+        vip: req.body.vip,
+        volunteerlist: req.body.volunteerlist,
+        newsletter: req.body.newsletter,
+      },
+
```

Comment:
I also think the creation of a contact is handled on line 77 with the call to `updateContact`. The double creation might also be the cause of the e2e test failures.